test/SCRStub.rb

```diff
@@ -58,3 +58,25 @@ def stub_scr_read(path_name)
     allow(Yast::SCR).to receive(:Read).with(path).and_return info
   end
 end
+
+module NetworkInterfacesStub
```

Comment:
You are right, firstly it was only for Network Interfaces but then i also stub other parts so i will change it ;)